src/collection-browser.ts

```diff
@@ -1088,27 +1088,6 @@ export class CollectionBrowser
     const sortParams = this.sortParam ? [this.sortParam] : [];
     const params: SearchParams = {
       query: this.fullQuery,
-      fields: [
```

Comment:
Q: where did this go?

Reply:
The fields param is no longer necessary since the PPS responses have all of these fields by default. But I think this change has since become part of another PR, so I'll undo the change here.